spec/services/reminders/update_service_spec.rb

```diff
@@ -41,7 +41,8 @@
 
     let(:model_instance) { create(:reminder, :scheduled, :with_unread_notifications, creator: user) }
     let(:user) { create(:admin) }
-    let(:call_attributes) { { remind_at: 2.days.from_now } }
+    let(:remind_at) { 2.days.from_now.change(hour: 12, min: 0) }
```

Comment:
Q: Curious if it was possible to generate a failing test case. AFAICT, changing the call attributes from a single "remind_at" to "remind_at_time" and "remind_at_date" should yield the same result- as the [Reminders::SetAttributesService](https://github.com/opf/openproject/blob/4fb720da4404bb816e8325dca057c15f6f29f198/app/services/reminders/set_attributes_service.rb#L34-L36) accepts either 🤔 Unless I'm missing sth